canonical-web-and-design/webteam-hubot
# Add /tz command

## Done

- Add `/tz` command to display timezone information for the given user(s)

## QA

- Run `@webbot tz @<some-username>` and verify it displays the timezone and current time for the user

## Screenshots

![tz-2](https://user-images.githubusercontent.com/995051/136050114-b2b46e58-3fee-45f4-b052-c046ac6a2027.png)
